mtcpclient/reporting.go

```diff
@@ -1,37 +1,38 @@
 package mtcpclient
 
 import (
-	"github.com/dachad/check-max-tcp-connections/tcpclient"
 	"fmt"
+	"github.com/dachad/check-max-tcp-connections/tcpclient"
 	"time"
 )
 
 func collectConnectionsStatus(connectionDescriptions []tcpclient.Connection, statusChannel <-chan tcpclient.Connection) {
 	for {
 		connectionStatus := <-statusChannel
 		connectionDescriptions[connectionStatus.Id] = connectionStatus
```

Comment:
question regarding this, I understand that we are updating the connectionStatus, but actually we are updating the Connection struct, including the ID and the connection status. Even it works, it's not fully clear to understand what it is actually doing. WDYT?

Reply:
the connection struct is exposed to the upper layer (from tcpclient to mtcpclient, which is consuming it), but its structure should remain as hidden as possible. The type connectionstatus is actually not exposed (now fixing this to become more clear), so I'd avoid using the internal members of the connection struct as much as possible... I'm renaming a var to see if we avoid mixing internal representation by how that is exposed to the upper layers